FirebaseFunctions/Tests/Unit/SerializerTests.swift

```diff
@@ -198,7 +198,7 @@ class SerializerTests: XCTestCase {
       1 as Int32,
       "two",
       [3 as Int32, ["@type": "type.googleapis.com/google.protobuf.Int64Value",
-                    "value": "9876543210"]],
+                    "value": "9876543210"]] as [Any],
```

Comment:
Are this and the other similar changes related to the weak specifiers or separate?

Reply:
This is a new Xcode 14.3 warning I noticed.